.github/workflows/tests.yml

```diff
@@ -0,0 +1,171 @@
+name: Run Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 1.6.x # remove when we create 'main'
+    tags-ignore: [ v.* ]
+
+
+jobs:
+  tests:
+    name: Build and Test
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { java-version: adopt@1.8,       scala-version: 2.12.13 }
+          - { java-version: adopt@1.8,       scala-version: 2.13.5 }
+          - { java-version: adopt@1.11.0-11, scala-version: 2.12.13 }
+          - { java-version: adopt@1.11.0-11, scala-version: 2.13.5 }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Checkout GitHub merge
+        if: github.event.pull_request
+        run: |-
+          git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
+          git checkout scratch
+
+      - name: Setup JDK ${{ matrix.java-version }}
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: ${{ matrix.java-version }}
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v5
+
+      - name: Run tests with Scala ${{ matrix.scala-version }} and Java ${{ matrix.java-version }}
+        run: sbt "++${{ matrix.scala-version }} test"
```

Comment:
I'm not sure why, but I've had problems with having `++...` in the quotes.

If you use `+~` instead, you get away with specifying the minor version `2.12` and `2.13`, the patch version is picked from the sbt files.

Reply:
Ah, interesting. I will change it.